src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -51,8 +51,9 @@ export const MapRoom: React.FC<MapRoomProps> = ({
   }, [dispatch]);
 
   const containerClasses = classNames("maproom", {
-    "maproom--covert--unclickable": isUnclickable,
-    "maproom--covert--iframe": isMapFrame,
+    "maproom--covert": isCovertRoom,
+    "maproom--unclickable": isUnclickable,
+    "maproom--iframe": isMapFrame,
     "maproom--always-show-label":
```

Comment:
Equivalent code location in `staging`. This likely came from a bad merge/similar:

- https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/PartyMap/components/Map/MapRoom.tsx#L53-L61